fix_checkpoint_history.py

```
@@ -0,0 +1,80 @@
+#!/usr/bin/env python3
+"""Fix checkpoint history to resume from existing CSV file"""
+
+import json
+from pathlib import Path
+from datetime import datetime
+
+def main():
```

Ruff (0.12.2)

Missing return type annotation for public function `main`

Add return type annotation: `None`

(ANN201)